stellar/gateway.js

```diff
@@ -188,6 +189,7 @@ if (require.main === module) {
         .addOption(new Option('--signers <signers>', 'Either use `wallet` or provide a JSON with the new signer set'))
         .addOption(new Option('--currentNonce <currentNonce>', 'nonce of the existing signers'))
         .addOption(new Option('--newNonce <newNonce>', 'nonce of the new signers (useful for test rotations)'))
+        .addOption(new Option('--estimateCost', 'estimate on-chain resources').default(false))
```

Comment:
addBaseOptions below adds this I think?

Reply:
Good call, moved it under `addBaseOptions`.